### PR TITLE
[NXP][rw61x][k32w1] Rework reference apps

### PR DESCRIPTION
**Pull request by Martin-NXP**
Changelog:

- Add multiple components for the app common code base
- Switch rw61x filesystem to NVS
- Add new rw61x revision board support
- New rw61x factory data encryption implementation
- Bump nxp_matter_support repo
- Bump ot-nxp repo
- Other bug fixes

### CODE REVIEW

**Comment by Martin-NXP** [comment]
Hello,
Is it possible to add the submodules update label please as it is intentional ?
Thank you

**Review by chapongatien — APPROVED** [review]

File: .github/workflows/examples-nxp.yaml [hunk 1]
```diff
@@ -150,7 +150,7 @@ jobs:
         if: github.actor != 'restyled-io[bot]'
 
         container:
-            image: ghcr.io/project-chip/chip-build-rw61x:74
```
Comment (andy31415) [inline .github/workflows/examples-nxp.yaml:153]: why the version rollback? could we use a -nxp 74 build?

I generally would not expect a version rollback in any circumstance. If we need to fix some build image, we should fix it and roll the version forward.

**Review by andy31415 — CHANGES_REQUESTED** [review]
Request changes: do not move backward in docker image versions.

File: .github/workflows/examples-nxp.yaml (hunk 1 above)
Comment (Martin-NXP) [reply .github/workflows/examples-nxp.yaml:153]: Hi Andy,
Thanks for the comment, I fix the docker image version

**Review by andy31415 — APPROVED** [review]

**Pull request merged**